recipes/cufflinks/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "cufflinks" %}
+{% set version = "0.12.1" %}
+{% set sha256 = "8f11e4b6326cc5b2a18011c09fb64f178ba21002f337fd305f64818012a6c679" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - plotting
```

Comment:
I don't think this package currently exists. Did this build on your local? Could this maybe be plotly (or not needed?)